etc/make/local.mk

```diff
@@ -20,12 +20,16 @@ STATICCHECK_FLAGS := \
 # this is currently greater than the number of examples tests
 EXAMPLES_JOBS ?= 16
 
-GEN_BINS_INTERNAL = $(BIN)/thriftrw-plugin-yarpc $(BIN)/protoc-gen-yarpc-go $(BIN)/v2/protoc-gen-yarpc-go
+GEN_BINS_INTERNAL = $(BIN)/thriftrw-plugin-yarpc $(BIN)/thriftrw-plugin-yarpc2 $(BIN)/protoc-gen-yarpc-go $(BIN)/v2/protoc-gen-yarpc-go
 
 $(BIN)/thriftrw-plugin-yarpc: ./encoding/thrift/thriftrw-plugin-yarpc/*.go
 	@mkdir -p $(BIN)
 	go build -o $(BIN)/thriftrw-plugin-yarpc ./encoding/thrift/thriftrw-plugin-yarpc
 
+$(BIN)/thriftrw-plugin-yarpc2: ./v2/yarpcthrift/thriftrw-plugin-yarpc2/*.go
```

Comment:
nit: Could we change this to `$(BIN)/v2/thritftw-plugin-yarpc` so that it's consistent with the `protoc-gen-yarpc-go` plugin?